modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -48,31 +50,36 @@
 }
 
 // Sidebar
-.sidebar-wrapper {
-	height: 100%;
-	width: 0;
 
-	&.open {
-		width: 320px;
-	}
-}
+$sidebarWidth: 320px;
```

Comment:
Personally I don't dislike having the variable in the same file of the style but the default pattern of Clay expect to have it in a dedicated one